src/core_plugins/input_control_vis/public/control/list_control_factory.js

```diff
@@ -7,9 +7,10 @@ const termsAgg = (field, size, direction) => {
     size = 1;
   }
   const terms = {
-    'size': size,
-    'order': {
-      '_count': direction
+    size: size,
+    shard_size: 10,
```

Comment:
I don't think setting `shard_size` is necessary in this context. It'll [be ignored](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#_shard_size_3) if the user picks a size greater than 10 and for sizes under 10 it probably doesn't make much of a difference. `timeout` and `terminate_after` are the real safeguards.